smtp/smtpc.c

```diff
@@ -241,6 +241,14 @@ smtpc_request_new(struct smtpc_env *env, const char *url, const char *from)
 		return NULL;
 	}
 	memset(req, 0, sizeof(*req));
+	req->error = (char *) malloc(CURL_ERROR_SIZE + sizeof(" (SMTP error)"));
```

Comment:
C doesn't require explicit casts. So it could be just `req->error = malloc(...)`.

BTW if you plan to use `memset(req, 0, sizeof(*req->error))` why don't you use calloc.

Reply:
Thank you for idea, fixed.

Reply:
Thank you for idea, fixed.